WordPressVIPMinimum/Sniffs/Functions/CreateFunctionSniff.php

```diff
@@ -0,0 +1,83 @@
+<?php
+/**
+ * WordPressVIPMinimum Coding Standard.
+ *
+ * @package VIPCS\WordPressVIPMinimum
+ */
+
+namespace WordPressVIPMinimum\Sniffs\Functions;
+
+use PHP_CodeSniffer_File as File;
+use PHP_CodeSniffer_Tokens as Tokens;
+
+/**
+ * This function generates an error when
+ * create_function() is found in code.
+ * create_function() is deprecated in PHP 7.2.0.
+ *
+ * An example:
+ *
+ * <code>
+ *   create_function( 'foo', 'return "bar";' );
+ * </code>
+ *
+ * See here: http://php.net/manual/en/function.create-function.php
+ */
+class CreateFunctionSniff implements \PHP_CodeSniffer_Sniff {
+	/**
+	 * Returns the token types that this sniff is interested in.
+	 *
+	 * We want everything function-related.
+	 *
+	 * @return array(int)
+	 */
+	public function register() {
+		return Tokens::$functionNameTokens;
+
+	}//end register()
+
+	/**
+	 * Processes the tokens that this sniff is interested in.
+	 *
+	 * @param \PHP_CodeSniffer\Files\File $phpcsFile The file where the token was found.
+	 * @param int                         $stackPtr  The position in the stack where
+	 *                                               the token was found.
+	 *
+	 * @return void
+	 */
+	public function process( File $phpcsFile, $stackPtr ) {
+		$this->_tokens    = $phpcsFile->getTokens();
+		$this->_phpcsFile = $phpcsFile;
+		$this->_stackPtr  = $stackPtr;
+
+		if (
+			( 'T_STRING' !==
+				$this->_tokens[ $this->_stackPtr ]['type'] )
+			||
+			( 'create_function' !==
+				$this->_tokens[ $this->_stackPtr ]['content'] )
+		) {
+			return;
+		}
+
+		$t_item_key = $this->_phpcsFile->findNext(
+			array( T_OPEN_PARENTHESIS ),
```

Comment:
I personally would look up the very next non-whitespace token and checked whether it's a bracket or not. By looking specifically for open bracket within current statement, the `\PHP_CodeSniffer\Files\File::findNext` method may match following code, for instance:

```
class Foo {
    public create_function = 'foo';
}
$c = new Foo();
$c->create_function = 'bar' . ( 1 === 1 ?? 'foo' );
```

Definitely an edge case, but I personally find the approach used eg. in https://github.com/squizlabs/PHP_CodeSniffer/blob/a9d8185ed9a23a981ca8bd698594b67f21a60e90/src/Standards/MySource/Sniffs/PHP/ReturnFunctionValueSniff.php#L47,L50 more robust.

btw: it would be good to add the code I shared above, or similar, to the testing suite, since it might help with further maintenance of this code, as it hardens the implementation of this sniff.